### ISSUE TITLE
[Request] Airlines connectors

### ISSUE THREAD

**Issue by Phyks** [cozy-labs/konnectors#478]
Similar to what is done with SNCF and Trainline, it would be useful to have connectors for airlines (AirFrance, Vueling, EasyJet, RyanAir, ...).

AirFrance (/FlyingBlue) may have an API to call https://developer.airfranceklm.com/page/Reservation_API, but I did not yet figure out how to handle it.

**Comment by Phyks** [comment]
So, typical airlines I am using and would be ok to work to make them appear in konnectors are:
- [ ] AirFrance
- [ ] EasyJet
- [ ] Vueling
- [ ] RyanAir

They are quite easy to handle I think, except for AirFrance. Indeed, AirFrance is having ReCaptcha on their connection form. And there does not seem to be any usable API. This is also the case on [the mobile website](https://mobile.airfrance.fr/FR/fr/local/login/login.do).

@ZeHiro Do you have any idea regarding this issue?

**Comment by ZeHiro** [comment]
It is a great idea.

**Comment by Phyks** [comment]
Sorry if it was confusing, "this issue" was referring to bypassing recaptcha, not the whole Github issue :)

**Comment by ZeHiro** [comment]
Oh yeah ok. Is there a captcha when connecting to airfrance account,?

**Comment by Phyks** [comment]
Yes captcha is needed to connect to airfrance account, both on the normal website and mobile website. :/
